Rakefile

```diff
@@ -1,9 +1,28 @@
 require 'rake'
 require 'rake/testtask'
 
-desc 'rake with no argument will run "rake test"'
-task default: :test
+require_relative 'lib/tasks/exercise'
+require_relative 'lib/tasks/exercise_test_tasks'
+require_relative 'lib/tasks/rake_arguments'
```

Comment:
Hmm, how should that work then?

Should `exercise_test_tasks` require them, even though it doesn't use any of those classes?